.eslintrc

```diff
@@ -53,7 +53,7 @@
     "max-len": [2, 80, 4, {
       "ignoreComments": true,
       "ignoreUrls": true,
-      "ignorePattern": ""
+      "ignorePattern": "^\\s*import\\s+{[^}]+}\\s+from\\s+'[^']+'\\s*;$"
```

Comment:
I don't believe this takes into account other forms of `import` like bare module imports:

```js
import 'some-really-long-path-to-my-really-long-module-name-that-exceeds-eighty-cars';
```

Reply:
"^\\s*import.*;"